default/python/AI/AIDependencies.py

```diff
@@ -399,6 +406,22 @@
     "SHP_SOLAR_CONT",
 )
 
+
+# Industry modifiers per population - [[EARLY_PRIORITY]]: Affected by species modifiers
+INDUSTRY_EFFECTS_PER_POP_MODIFIED_BY_SPECIES = {
```

Comment:
I don't understand what you mean with this. 

What should that function return and when would I call it? What tags are you talking about?